x-pack/plugins/rule_registry/common/assets/field_maps/technical_rule_field_map.test.ts

```diff
@@ -43,9 +43,6 @@ it('matches snapshot', () => {
       "kibana.alert.end": Object {
         "type": "date",
       },
-      "kibana.alert.flapping": Object {
```

Comment:
It's not used anymore?

Reply:
This was just added [recently by the framework](https://github.com/elastic/kibana/pull/143920) so we'd like to keep it :)